hack/validate-vendor.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+####################################################
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+####################################################
+
+set -x
+
+T="$(mktemp -d)"
+trap "rm -rf ${T}" EXIT
+
+mkdir -p $T/src/github.com/openshift/openshift-azure
+cp -a "${DIR}/.." "${T}/src/github.com/openshift/openshift-azure"
+
+GOPATH=$T go build github.com/openshift/openshift-azure/...
```

Comment:
How is this different from [`make build`](https://github.com/openshift/release/blob/f80e62f1b8ea73ef4b4c0951e1603777cf80c53d/ci-operator/config/openshift/openshift-azure/master.json#L24) that we already run in CI?

Reply:
Agreed - I thought I must be missing something.
@mjudeikis I think you were a bit fast on the lgtm button there ;)
Will revert.